Core/GraphQL/GraphQLManager.swift

```diff
@@ -50,9 +51,13 @@ extension GraphQLManager: GraphQLManageable {
 
     public func dispatchMutation<Mutation: GraphQLMutation>(
         mutation: Mutation,
+        cacheResultToPersistence: Bool,
```

Comment:
May I know why we don't need `cacheResultToPersistence` for  `dispatch<Query: GraphQLQuery>` function please? Thank you.

Reply:
No problem at all.

The interfaces of fetching query and performing mutation in `ApolloClient` are different, so the arguments of the two functions in GraphQLManager are different too.

```
public func fetch<Query: GraphQLQuery>(query: Query,
  cachePolicy: CachePolicy = .returnCacheDataElseFetch,
  contextIdentifier: UUID? = nil,
  queue: DispatchQueue = DispatchQueue.main,
  resultHandler: GraphQLResultHandler<Query.Data>? = nil) -> Cancellable

public func perform<Mutation: GraphQLMutation>(mutation: Mutation,
  publishResultToStore: Bool = true,
  queue: DispatchQueue = .main,
  resultHandler: GraphQLResultHandler<Mutation.Data>? = nil) -> Cancellable
```

The cache policy is more flexible on the query in the Apollo SDK.

Reply:
Got it, thank you for your explanation!